code/game/objects/items/blueprints.dm

```diff
@@ -112,7 +112,7 @@
 
 
 /obj/item/areaeditor/proc/get_area_type(area/A = get_area())
```

Comment:
unrelated but this proc is a monstrosity

hell all the procs for this path are, really